modules/osdk-upgrading-v180-to-v1100.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-upgrading-projects.adoc
+
+:osdk_ver: v1.10.1
+:osdk_ver_n1: v1.8.0
+
+[id="osdk-upgrading-v180-to-v1100_{context}"]
+= Upgrading projects for Operator SDK {osdk_ver}
+
+The following upgrade steps must be performed to upgrade an existing Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- Operator project that was previously created or maintained with Operator SDK {osdk_ver_n1}
+- If the project was created with a version prior to {osdk_ver_n1}, completion of the link:https://docs.openshift.com/container-platform/4.8/operators/operator_sdk/osdk-upgrading-projects.html#osdk-upgrading-v130-to-v180_osdk-upgrading-projects[Upgrading projects for Operator SDK v1.8.0] procedure 
+
+.Procedure
+
+* Ansible projects are now scaffolded with Kustomize version 3.8.7. This version of Kustomize requires that the path to patch files be provided with the `--path` flag in the `add patch` command.
```

Comment:
Please move the action to the start of the step - it's more helpful to tell the user what they're doing first and why they're doing it after. Are you sure that your user is going to know how to complete this action without more context?